### PR TITLE
Remove need for JSON file.

### PR DESCRIPTION
**Pull request by trentwiles**
JSON files with API details can be a bit trick as you do not want them public on the internet. I removed the need for one to simplify the code.

### CODE REVIEW

**Comment by Xwilarg** [comment]
Thanks for the PR!
The JSON part was really just for the example but I agree that it might be indeed a bit confusing.
However I'm not so sure about removing the comments explaining what the différents elements are?

**Comment by trentwiles** [comment]
Oh, sorry about that. I could revise it to include the old comments.

**Comment by trentwiles** [comment]
There, done.

File: sample/index.php [hunk 1]
```diff
@@ -4,12 +4,12 @@
 
 use Xwilarg\Discord\OAuth2;
 
-// Sample JSON configuration file, contains the following fields:
 // clientId: Client ID of the application
 // secret: Secret of the application
 // url: The redirect URL (URL called after the user is logged in, must be registered in https://discordapp.com/developers/applications/[YourAppId]/oauth)
-$auth = json_decode(file_get_contents('Credentials.json'), true);
-$oauth2 = new OAuth2($auth["clientId"], $auth["secret"], $auth["url"]);
+
```
Comment (Xwilarg) [inline sample/index.php]: Could you also please remove this empty line and change the name in the comments to match the new argument names?

**Review by Xwilarg — CHANGES_REQUESTED** [review]

**Comment by trentwiles** [comment]
Added. Does this work?

File: sample/index.php (hunk 1 above)
Comment (trentwiles) [reply sample/index.php]: Added.

**Comment by Xwilarg** [comment]
Great, thanks a lot!

**Pull request merged**